### DISCUSSION TITLE
Album Options

### DISCUSSION THREAD

**Discussion by tinohager** [LycheeOrg/Lychee#2610] [Bugs] [answered]
### Lychee version

v6.0.0

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

When I delete an album and then switch to another album, I don't end up in the photo overview but directly in the settings dialog of the album. The same happens when I create a new album after deleting another album.

### Steps to reproduce the issue

1. Goto to Album
2. Open the Album Settings
3. Switch to Danger Zone
4. Delete the Album and Photos
5. Open an other Album

### Diagnostics [REQUIRED]

If I post something here, there should be a button in the Diagnostic section where I can copy all the information to the clipboard. 

We have Errors, Info, Space, Configuration, ... there.

### Browser & System [REQUIRED]

Chrome, docker:latest

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
fixed: https://github.com/LycheeOrg/Lychee/pull/2684
